etc/kube/pachyderm.json

```diff
@@ -279,7 +279,7 @@
     }
   },
   "spec": {
-    "replicas": 8,
```

Comment:
Is this change made in the deploy binary as well? This file is actually auto generated.